src/test/java/org/alliancegenome/curation_api/base/BaseITCase.java

```diff
@@ -135,39 +135,32 @@ public void checkFailedBulkLoad(String endpoint, String filePath, HashMap<String
 			resp.body("history.counts." + entry.getKey() + ".total", is(entry.getValue().get("total")));
 			resp.body("history.counts." + entry.getKey() + ".failed", is(entry.getValue().get("failed")));
 			resp.body("history.counts." + entry.getKey() + ".completed", is(entry.getValue().get("completed")));
+			resp.body("history.counts." + entry.getKey() + ".skipped", is(entry.getValue().get("skipped")));
 		}
 	}
-	
 
-	public void checkSuccessfulBulkLoad(String endpoint, String filePath) throws Exception {
-		checkSuccessfulBulkLoad(endpoint, filePath, "Records", 1);
+	public void checkSkippedBulkLoad(String endpoint, String filePath) throws Exception {
+		checkBulkLoadRecordCounts(endpoint, filePath, "Records", 1, 0, 0, 1);
 	}
 	
-	public void checkSuccessfulBulkLoad(String endpoint, String filePath, int nrRecords) throws Exception {
-		checkSuccessfulBulkLoad(endpoint, filePath, "Records", nrRecords);
+	public void checkSkippedBulkLoad(String endpoint, String filePath, int nrRecords) throws Exception {
+		checkBulkLoadRecordCounts(endpoint, filePath, "Records", nrRecords, 0, 0, nrRecords);
 	}
-
-	public void checkSuccessfulBulkLoad(String endpoint, String filePath, String countType, int nrRecords) throws Exception {
-		String content = Files.readString(Path.of(filePath));
-
-		RestAssured.given().
-			contentType("application/json").
-			body(content).
-			when().
-			post(endpoint).
-			then().
-			statusCode(200).
-			body("history.counts." + countType + ".total", is(nrRecords)).
-			body("history.counts." + countType + ".failed", is(0)).
-			body("history.counts." + countType + ".completed", is(nrRecords));
+	
+	public void checkSuccessfulBulkLoad(String endpoint, String filePath) throws Exception {
+		checkSuccessfulBulkLoad(endpoint, filePath, 1);
 	}
 	
+	public void checkSuccessfulBulkLoad(String endpoint, String filePath, int nrRecords) throws Exception {
+		checkBulkLoadRecordCounts(endpoint, filePath, "Records",nrRecords, 0, nrRecords, 0);
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChA5MmJhY2Y2OWExOTk5ZjMwEgpjaGVja3N0eWxl -->